### ISSUE TITLE
Login with 2 factor auth can fail

### ISSUE THREAD

**Issue by Tunous** [slapperwan/gh4a#522]
OctoDroid tries to handle the https://github.com/sessions/two-factor link instead of opening it in browser.

**Comment by maniac103** [comment]
Is this triggered by us or do you mean it's incorrectly handled when opened in browser?

**Comment by Tunous** [comment]
I think both but what happened is this:

I started the app and pressed the login button. It showed a popup to select in which application to handle the login and I selected OctoDroid. It then redirected me to the login screen and here I logged in. After that, another popup showed up where I one more time selected OctoDroid but this time instead of redirecting me to the browser it treated the link as a repository and I was presented with an error screen that had 'sessions/two-factor' in the toolbar.
